app/components/checkboxes/none-of-the-above.njk

```diff
@@ -0,0 +1,108 @@
+{% set html_style = 'background-color: #f0f4f5;' %}
+{% set title = 'Checkboxes with conditional content' %}
```

Comment:
Very minor but page title just needs to be `Checkboxes with "none of the above" option`